test/types/file/unableToOpenFile.good

```diff
@@ -1 +1 @@
-unableToOpenFile.chpl:1: error: No such file or directory in open with path "_test_cannotOpenMe.txt"
+unableToOpenFile.chpl:1: error: uncaught error - No such file or directory in open with path "_test_cannotOpenMe.txt"
```

Comment:
Let's call what happens in this PR "good enough for now" and open an issue to talk about the exact details of the format, wherein we can discuss further the Java analogue, original line numbers, etc. I do have more to say, so @ mention me when you make the issue.